docs/source/using-diffusers/loading.mdx

```diff
@@ -12,7 +12,347 @@ specific language governing permissions and limitations under the License.
+- *Schedulers* and *Configurations* via [`ConfigMixin.from_config`]
+
+## Loading pipelines
+
+The [`DiffusionPipeline`] class is the easiest way to access any diffusion model thas is [available on the Hub](https://huggingface.co/models?library=diffusers). Let's look at an example on how to download [CompVis' Latent Diffusion model](https://huggingface.co/CompVis/ldm-text2im-large-256).
+
+```python
+from diffusers import DiffusionPipeline
+
+repo_id = "CompVis/ldm-text2im-large-256"
+ldm = DiffusionPipeline.from_pretrained(repo_id)
+```
+
+Here [`DiffusionPipeline`] automatically detects the correct pipeline being [`LDMTextToImagePipeline`], downloads and caches all required configuration and weight files if not already done so, and finally returns a pipeline instance `ldm`.
+The pipeline instance can then be called using [`LDMTextToImagePipeline.__call__`] for text-to-image generation.
+
+Instead of using the generic [`DiffusionPipeline`] class for loading, one can also directly use the fitting pipeline class if known by the user. The code snippet above yields the same instance as when doing:
+
+```python
+from diffusers import LDMTextToImagePipeline
+
+repo_id = "CompVis/ldm-text2im-large-256"
+ldm = LDMTextToImagePipeline.from_pretrained(repo_id)
+```
+
+Diffusion pipelines like `LDMTextToImagePipeline` often consist of multiple components. These components conists both of parameterized models, such as `"unet"`, "vqvae"` and "bert", tokenizers and schedulers which interact sometimes in complex way with each other, *e.g.* for [`LDMTextToImagePipeline`] or [`StableDiffusionPipeline`] the pipeline is explained [here](https://huggingface.co/blog/stable_diffusion#how-does-stable-diffusion-work).
+The purpose of the [pipeline classes](./api/overview#diffusers-summary) is to wrap the complexity of these pipeline systems and give the user the most easy-to-use API while staying flexible for customization as will be shown later.
+
+### Loading pipelines that require access request
+
+Due to the capabilities of diffusion models to generate extremely realistic images, there is a certain danger that such models might be misused for unwanted applications, *e.g.* generating pornography of violent images.
+In order to limit the possibility of such unwanted use cases, some of the most powerful diffusion models require users to acknowledge a license before being able to use the model or else the pipeline cannot be downloaded.
+If you try to load [`runwayml/stable-diffusion-v1-5`](https://huggingface.co/runwayml/stable-diffusion-v1-5) the same way as:
+
+```python
+from diffusers import DiffusionPipeline
+
+repo_id = "runwayml/stable-diffusion-v1-5"
+stable_diffusion = DiffusionPipeline.from_pretrained(repo_id)
+```
+
+it will only work if you have both *click-accepted* the license on [the model card](https://huggingface.co/runwayml/stable-diffusion-v1-5) and if you're logged into the Hugging Face Hub. Otherwise you will get an error message
```

Comment:
```suggestion
it will only work if you have both *click-accepted* the license on [the model card](https://huggingface.co/runwayml/stable-diffusion-v1-5) and are logged into the Hugging Face Hub. Otherwise you will get an error message
```